app/graphql/schema.rb

```diff
@@ -15,6 +15,5 @@ class Schema < GraphQL::Schema
   mutation Types::Mutation
   lazy_resolve(Promise, :sync)
   use GraphQL::Batch
-  use GraphQL::Execution::Interpreter
-  use GraphQL::Analysis::AST
+  use GraphQL::Cache
```

Comment:
Unfortunately the new interpreter is incompatible with gql-cache (https://graphql-ruby.org/queries/interpreter.html)  (https://github.com/stackshareio/graphql-cache/issues/71)